### PR TITLE
s2i-enable

### PR DESCRIPTION
**Pull request by TomasTomecek**
Signed-off-by: Tomas Tomecek <ttomecek@redhat.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,6 +56,59 @@ $ docker run -P \
...
+$ sudo docker run new-vsftpd
++ set -o nounset
++ set -o pipefail
++ '[' -f ./vsftpd.conf ']'
++ CONF_FILE=./vsftpd.conf
++ '[' -f ./init-hook ']'
++ source ./init-hook
+++ echo 'You can put your custom logic here'
+You can put your custom logic here
++ exec /usr/sbin/vsftpd ./vsftpd.conf
```
Comment (dhodovsk) [inline README.md]: When I tested it ```docker run``` wrote only output of the commands specified above, that means one line
```
You can put your custom logic here
```
Could you please take look at this?

**Comment by dhodovsk** [comment]
One nit, otherwise LGTM

File: README.md (hunk 1 above)
Comment (TomasTomecek) [reply README.md]: You are correct, the readme content is not up to date. I will update.

**Comment by TomasTomecek** [comment]
I updated readme and tried to describe the extending process more clearly. Can you take a look?

**Pull request merged**